docs/_data/components/schemas/time-bound-static-ratings.yaml

```diff
@@ -37,28 +36,53 @@ temporary-aar-exception: &time-bound-static-limit
   <<: *time-bound-static-rating
   description: |
     Data structure for a temporary AAR Exception against a resource.
-    Includes a unique ID, start and (optional) end time, and a reason.  
+    Includes a unique ID, start and (optional) end time, and a reason.
 
     May or may not include a set of values, depending on the business
-    context in which the temporary AAR exception is used.  
+    context in which the temporary AAR exception is used.
   properties: &time-bound-static-limit-props
     <<: *time-bound-static-rating-props
     id:
       $ref: ./server-generated-id.yaml
   additionalProperties: false
   required:
-    - source
```

Comment:
My rationale here is that a `temporary-aar-exception` might originate with the clearinghouse itself, so a `source` would be implicit.